src/main/resources/org/broadinstitute/hellbender/utils/config/GATKConfig.properties

```diff
@@ -0,0 +1,65 @@
+# ======================================================================================================================
+# System Properties
+#
+# All Configuration options in ths section are to be injected into the JRE
+# as System Properties.
+#
+# Any property beginning with `system.` (without quotes) will be injected into the JRE as a System Property.
+# Properties not starting with `system.` (without quotes) will NOT be used as System Properties.
+#=======================================================================================================================
+
+# ---------------------------------------------------------------------------------
+# General Options:
+# ---------------------------------------------------------------------------------
+
+system.gatk_stacktrace_on_user_exception = true
+
+# ---------------------------------------------------------------------------------
+# SAMJDK Options:
+# ---------------------------------------------------------------------------------
+
+system.samjdk.use_async_io_read_samtools = true
+system.samjdk.use_async_io_write_samtools = true
+system.samjdk.use_async_io_write_tribble = false
+system.samjdk.compression_level = 1
+
+# ----------------------------------------------------------
+# Spark Options:
+# ----------------------------------------------------------
+
+system.spark.kryoserializer.buffer.max = 512m
+system.spark.io.compression.codec = lzf
+system.spark.yarn.executor.memoryOverhead = 600
+
+system.spark.driver.maxResultSize = 0
+system.spark.driver.userClassPathFirst = true
+system.spark.driver.extraJavaOptions =
```

Comment:
It looks like those empty values will only work in cases where the property/config type is a string.  

I haven't looked, but I'm guessing this would be a pretty simple fix in OWNER.  Alternatively we could make custom decoder classes for all basic types and just use those to be safe.  A second alternative would be to do nothing and release documentation that you need to trim whitespace and declare values for non-string types.